src/BenchmarkDotNet/Toolchains/DotNetCli/DotNetCliCommand.cs

```diff
@@ -126,5 +153,15 @@ private static string GetCustomMsBuildArguments(BenchmarkCase benchmarkCase, IRe
 
             return string.Join(" ", msBuildArguments.Select(arg => arg.TextRepresentation));
         }
+
+        private static IEnumerable<string> GetNugetAddPackageCommands(BenchmarkCase benchmarkCase, IResolver resolver)
+        {
+            if (!benchmarkCase.Job.HasValue(InfrastructureMode.NugetReferencesCharacteristic))
+                return Enumerable.Empty<string>();
+
+            var nugetRefs = benchmarkCase.Job.ResolveValue(InfrastructureMode.NugetReferencesCharacteristic, resolver).OfType<NugetReference>();
```

Comment:
is the `OfType<NugetReference>()` required here?

Reply:
Nope, i think that's copy/paste fail :)